website/docs/r/notification_destination.html.markdown

```diff
@@ -36,30 +36,33 @@ See additional [examples](#additional-examples).
+* `account_id` - (Required) Determines the New Relic account where the notification destination will be created. Defaults to the account associated with the API key used.
 * `name` - (Required) The name of the destination.
 * `type` - (Required) The type of destination.  One of: `EMAIL`, `SERVICE_NOW`, `WEBHOOK`, `JIRA`, `PAGERDUTY_ACCOUNT_INTEGRATION` or `PAGERDUTY_SERVICE_INTEGRATION`.
-* `auth` - A nested block that describes a notification destination authentication. Only one auth block is permitted per notification destination definition.  See [Nested auth blocks](#nested-auth-blocks) below for details.
-* `properties` - A nested block that describes a notification destination properties. See [Nested properties blocks](#nested-properties-blocks) below for details.
+* `auth_basic` - (Optional) A nested block that describes a basic username and password authentication credentials. Only one auth_basic block is permitted per notification destination definition.  See [Nested auth_basic blocks](#nested-auth_basic-blocks) below for details.
+* `auth_token` - (Optional) A nested block that describes a token authentication credentials. Only one auth_token block is permitted per notification destination definition.  See [Nested auth_token blocks](#nested-auth_token-blocks) below for details.
+* `property` - (Optional) A nested block that describes a notification destination property. See [Nested property blocks](#nested-property-blocks) below for details.
 
-### Nested `auth` blocks
+### Nested `auth_basic` blocks
 
-* `type` - (Required) The type of the auth.  One of: `TOKEN` or `BASIC`.
+* `user` - (Required) The username of the basic auth.
+* `password` - (Required) Specifies an authentication password for use with a destination.
 
-Each authentication type supports a specific set of arguments:
+### Nested `auth_token` blocks
 
-* `BASIC`
-  * `user` - (Required) The username of the basic auth.
-  * `password` - (Required) Specifies an authentication password for use with a destination.
-* `TOKEN`
-  * `prefix` - (Required) The prefix of the token auth.
-  * `token` - (Required) Specifies the token for integrating.
+* `prefix` - (Required) The prefix of the token auth.
+* `token` - (Required) Specifies the token for integrating.
 
 ~> **NOTE:** OAuth2 authentication type is not available via terraform for notifications destinations.
 
-### Nested `properties` blocks
+### Nested `property` blocks
 
-Each notification destination type supports a specific set of arguments for the `properties` block:
+* `key` - (Required) The notification property key.
+* `value` - (Required) The notification property value.
+* `label` - (Optional) The notification property label.
+* `display_value` - (Optional) The notification property display value.
+
+Each notification destination type supports a specific set of arguments for the `property` block:
```

Comment:
Can we add a link to any official documentation that shows what each type supports? If not, it would be good for us to either have a couple examples or for us to document which types support which values.

Reply:
I added a link to each of the destination types examples below (in the title), this is also the reason we added examples for each type.